symfony/src/Entity/Site.php

```diff
@@ -90,7 +90,7 @@ class Site
     /**
      * @ORM\Column(type="smallint")
```

Comment:
This was already here before, but since we're updating the Site entity, should we change this to a boolean type? You'll need to check where else we are using this and make any adjustments if necessary.

Reply:
The status column should probably be a boolean also.